ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -30,4 +61,37 @@
 <InfoTableRow @label="Default lease TTL" @value={{@mountConfig.config.defaultLeaseTtl}} />
 <InfoTableRow @label="Max lease TTL" @value={{@mountConfig.config.maxLeaseTtl}} />
 <InfoTableRow @label="Allowed managed keys" @value={{or @mountConfig.config.allowedManagedKeys "None"}} />
-<div class="has-top-margin-l"></div>
\ No newline at end of file
+<div class="has-top-margin-l"></div>
+
+{{#if this.showDeleteAllIssuers}}
+  <Modal
+    @title="Delete All Issuers?"
+    @type="warning"
+    @onClose={{fn (mut this.showDeleteAllIssuers) false}}
+    @isActive={{this.showDeleteAllIssuers}}
+    @showCloseButton={{true}}
+  >
+    <section class="modal-card-custom has-padding-m">
+      This endpoint deletes all issuers and keys within the mount. It is highly recommended to use the individual delete
+      operations instead. This mount will be unusable until new issuers and keys are provisioned.
+      <div class="has-top-margin-m">
+        <h4 class="title is-6 has-bottom-margin-xs">Confirm</h4>
+        <span class="is-hint">Type <b>delete-all</b> to confirm deleting the role.</span>
```

Comment:
I think this might be a typo in the designs, since this isn't a role we're deleting. 
```suggestion
        <span class="is-hint">Type <b>delete-all</b> to confirm deleting the role.</span>
```

Reply:
@hellobontempo what do you think of `Type <b>delete-all</b> to confirm deleting all issuers and keys.`?

Reply:
Lol I think that was what I had written in my "suggestion" but I must have deleted it 😅  accidentally - I think that's perfect!